usage/launcher/src/test/java/org/apache/brooklyn/launcher/BrooklynWebServerTest.java

```diff
@@ -146,11 +146,13 @@ public void verifyHttpsFromConfig() throws Exception {
     @Test
     public void verifyHttpsCiphers() throws Exception {
         brooklynProperties.put(BrooklynWebConfig.HTTPS_REQUIRED, true);
-        brooklynProperties.put(BrooklynWebConfig.TRANSPORT_PROTOCOLS, "XXX");
-        brooklynProperties.put(BrooklynWebConfig.TRANSPORT_CIPHERS, "XXX");
```

Comment:
I must confess I did not understand the intention of this test. Jetty9 is throwing `java.net.SocketException`. Shall we add that to the list? Or just replace the list with this exception?